### PR TITLE
:zap: | Webpage revamp

### PR DESCRIPTION
**Pull request by HUSKUT**
No tailwind because I genuinely cannot be arsed to make those buttons myself.

part 2 will come with a list soon ™️

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        if (navigator.appVersion.indexOf("Linux") != -1) os =
+            "Linux OS";
+
+        return os;
+    }
+
+    let os = getOS()
+    let oselement = document.getElementById("platform")
+
+    if (os === 'Windows') {
```
Comment (marnixah) [inline web/index.html:58]: This check will always fail, no?

Comment (HUSKUT) [reply web/index.html:58]: No, this actually works. Atleast for MacOS, have yet to try it for Windows/Linux. Although if it does not work, it won't break the website or server.

**Comment by HUSKUT** [comment]
Forgot to put main.go in the commit aswell, now that is added aswell. So this should work in a whole

File: web/index.html (hunk 1 above)
Comment (marnixah) [reply web/index.html:58]:
```js
os = "Windows OS";

if (os === 'Windows') {
```
How would this work?

Comment (HUSKUT) [reply web/index.html:58]: Shouldn't use windows then.

**Review by marnixah — APPROVED** [review]
😃😃😃

**Pull request merged**